client/stores/access_tokens.js

```diff
@@ -0,0 +1,42 @@
+import { defineStore } from "pinia"
+import { useContentStore } from "~/composables/stores/useContentStore.js"
+
+export const useAccessTokenStore = defineStore("access_tokens", () => {
+  const contentStore = useContentStore()
+
+  const abilities = [
```

Comment:
Since they're all required for Zapier, can we please make all the checkboxes enabled by default, and disable the checkbox so that they can't be unchecked. That way it would be less confusing for user to get started with Zapier. Whenever we add more integrations requiring tokens, or more features via the "external api" we can change how this works